python/taichi/lang/matrix.py

```diff
@@ -1900,6 +1904,11 @@ def __call__(self, *args):
                 entries += list(x.ravel())
             elif isinstance(x, Matrix):
                 entries += x.entries
+            elif isinstance(x, impl.Expr) and x.ptr.is_tensor():
```

Comment:
Should this also be added to `MatrixType`?

Reply:
thanks~